src/Nancy/Conventions/BuiltInCultureConventions.cs

```diff
@@ -133,7 +133,12 @@ public static CultureInfo CookieCulture(NancyContext context, GlobalizationConfi
         /// <returns>CultureInfo from CurrentThread</returns>
         public static CultureInfo ThreadCulture(NancyContext context, GlobalizationConfiguration configuration)
         {
+#if DOTNET5_4
+            CultureInfo.CurrentCulture = new CultureInfo(configuration.SupportedCultureNames.First());

```

Comment:
Is this even the same thing as `Thread.CurrentThread.CurrentCulture` maybe it is the `ThreadCulture` that should not be supported on `dotnet5_4` at all?